click_odoo_contrib/_dbutils.py

```diff
@@ -47,3 +47,17 @@ def db_management_enabled():
         yield
     finally:
         odoo.tools.config["list_db"] = list_db
+
+
+def reset_config_parameters(dbname):
+    """
+    Reset config parameters to default value. This is useful to avoid
+    conflicts between databases on copy or restore
+    (dbuuid, ...)
+    """
+    with OdooEnvironment(dbname) as env:
+        try:
+            env["ir.config_parameter"].init(force=True)
+        except TypeError:
+            # odoo < 10
+            env.registry("ir.config_parameter").init(env.cr, force=True)
```

Comment:
@lmignon a check on `odoo.release.version_info` would be more robust than catching `TypeError`.

Reply:
done